internal/util/parse.go

```diff
@@ -85,6 +87,14 @@ func ReadUintFromFile(path string) (uint64, error) {
 	if err != nil {
 		return 0, err
 	}
+	for _, c := range data {
+		if string(c) == "\n" {
+			continue
+		}
+		if !unicode.IsDigit(rune(c)) {
```

Comment:
Hmm, this is going to match a whole bunch of unicode glyph runes that are not `[0-9]`. I think we should be more strict and check like this:

 ```suggestion
		if !strings.ContainsAny(c, "0123456789") {
```